.github/workflows/docs.yml

```diff
@@ -14,24 +14,20 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
     - uses: actions/checkout@v4
+      with: 
+        submodules: true
     - name: Set up Python 3.11
       uses: actions/setup-python@v5
       with:
         python-version: '3.11'
-    - name: Upgrade pip and install wheel
-      run: pip install --upgrade pip wheel
-    - name: Install pytket qiskit
-      run: pip install .
+    - name: install poetry
+      run: pip install poetry
     - name: Install docs dependencies
-      run: |
-        pip install -r .github/workflows/docs/requirements.txt
+      run: cd docs && bash install.sh 
+    - name: Install pytket qiskit
+      run: cd docs && poetry run pip install ../.
     - name: Test building docs
-      timeout-minutes: 20
+      timeout-minutes: 10
```

Comment:
Have you reduced the timeout on purpose?

Reply:
Yes, would you prefer I changed it back?